python/pylibcudf/pylibcudf/io/csv.pyx

```diff
@@ -263,3 +274,199 @@ def read_csv(
+        return self
+
+    cpdef CsvWriterOptionsBuilder include_header(self, bool val):
+        """Enables/Disables headers being written to csv.
+
+        Parameters
+        ----------
+        val : bool
+            Boolean value to enable/disable
+
+        Returns
+        -------
+        CsvWriterOptionsBuilder
+            Builder to build CsvWriterOptions
+        """
+        self.c_obj.include_header(val)
+        return self
+
+    cpdef CsvWriterOptionsBuilder rows_per_chunk(self, int val):
+        """Sets maximum number of rows to process for each file write.
+
+        Parameters
+        ----------
+        val : int
+            Number of rows per chunk
+
+        Returns
+        -------
+        CsvWriterOptionsBuilder
+            Builder to build CsvWriterOptions
+        """
+        self.c_obj.rows_per_chunk(val)
+        return self
+
+    cpdef CsvWriterOptionsBuilder line_terminator(self, str term):
+        """Sets character used for separating lines.
+
+        Parameters
+        ----------
+        term : str
+            Character to represent line termination
+
+        Returns
+        -------
+        CsvWriterOptionsBuilder
+            Builder to build CsvWriterOptions
+        """
+        self.c_obj.line_terminator(term.encode())
+        return self
+
+    cpdef CsvWriterOptionsBuilder inter_column_delimiter(self, str delim):
+        """Sets character used for separating column values.
+
+        Parameters
+        ----------
+        delim : str
+            Character to delimit column values
+
+        Returns
+        -------
+        CsvWriterOptionsBuilder
+            Builder to build CsvWriterOptions
+        """
+        self.c_obj.inter_column_delimiter(ord(delim))
+        return self
+
+    cpdef CsvWriterOptionsBuilder true_value(self, str val):
+        """Sets string used for values != 0
+
+        Parameters
+        ----------
+        val : str
+            String to represent values != 0
+
+        Returns
+        -------
+        CsvWriterOptionsBuilder
+            Builder to build CsvWriterOptions
+        """
+        self.c_obj.true_value(val.encode())
+        return self
+
+    cpdef CsvWriterOptionsBuilder false_value(self, str val):
+        """Sets string used for values == 0
+
+        Parameters
+        ----------
+        val : str
+            String to represent values == 0
+
+        Returns
+        -------
+        CsvWriterOptionsBuilder
+            Builder to build CsvWriterOptions
+        """
+        self.c_obj.false_value(val.encode())
+        return self
+
+    cpdef CsvWriterOptions build(self):
+        """Moves the CSV writer options builder"""
+        cdef CsvWriterOptions csv_options = CsvWriterOptions.__new__(
+            CsvWriterOptions
+        )
+        csv_options.c_obj = move(self.c_obj.build())
+        csv_options.table = self.table
+        csv_options.sink = self.sink
+        return csv_options
+
+
+def write_csv(
```

Comment:
Yes. We've been cpdef'ing them, so I did it here too

Reply:
I think it's good practice to do it consistently everywhere to enable full Cython usage. For I/O it's probably less important in the grand scheme of things since that shouldn't propagate bad typing afterwards in any way, but consistency helps.